spec/preservation/update_moab_spec.rb

```diff
@@ -0,0 +1,45 @@
+describe Preservation::UpdateMoab do
+  let(:pres_update_moab) { described_class.new }
+  let(:full_druid) { 'druid:bj102hs9687' }
+  let(:mock_path) { instance_double(Pathname) }
+  let(:mock_so) { instance_double(Moab::StorageObject, object_pathname: mock_path) }
+
+  context '#perform' do
+    it 'calls #update_moab' do
```

Comment:
See my comment below  -- I think it might be slightly better to directly test that the perform action calls ingest_bag and verify_version_storage on the newly created Moab/Moab version.